### ISSUE TITLE
Success and Botch messages

### ISSUE THREAD

**Issue by rpgmik** [MrMstislav/DiscordDegeneSix#4]
On the DegeneSix bot running on the Degenesis discord server, it looks like the Success and Botch messages (which include the :degenesis: and :skull: emojis) have been removed from the results output.  Should the code be updated here to reflect that?

**Comment by rpgmik** [comment]
Sorry, on closer inspection, I see it depends on how users submit their rolls (i.e., whether they specify the number of successes required in addition to the number of dice to roll).  BUT, it looks like the :degenesis: output is not producing the correct emoji in that case.

**Comment by daviegourevitch** [comment]
@rpgmik are you using it on a server other than the degenesis server? Server specific emojis need to include the ID

**Comment by rpgmik** [comment]
@daviegourevitch - thanks for that.  Yes, I was using it on my own server, so that is now fixed. I had noticed the emojis failing to post on the main Degenesis discord server too though.  It looks like the emoji ID numbers changed at some point, so I've updated the code to reflect the new numbers.

**Comment by rpgmik** [comment]
Of course, for anyone inviting MrMstislav's bot (which runs on Amazon) to their own server, they won't ever be able to get the custom emoji's working, since even if they add the correct emojis, they won't have the same ID.  I don't know of a way to use the same emoji ID number across multiple servers, but perhaps it is possible?

**Comment by rpgmik** [comment]
Actually, I found a way to do it.  The bot can get the server-specific emoji ID (as long as it is named correctly), and use it to post the emoji.  See patch: https://github.com/MrMstislav/DiscordDegeneSix/pull/6

**Comment by daviegourevitch** [comment]
Oh sweet, that's really cool, didn't know that was possible
